src/breeze/components/breeze/sidebar_links.cr

```diff
@@ -1,21 +1,32 @@
 class Breeze::SidebarLinks < BreezeComponent
+  needs context : HTTP::Server::Context
+
   def render
-    # div class: "flex items-center flex-shrink-0 px-4" do
-    #   img alt: "Workflow", class: "h-8 w-auto", src: "/img/logos/workflow-logo-on-white.svg"
-    # end
     nav class: "mt-5 flex-1 px-2 bg-white" do
-      link to: Breeze::Requests::Index, class: "group flex items-center px-2 py-2 text-sm leading-5 font-medium text-green-100 rounded-md bg-green-400 hover:text-white hover:bg-green-500 focus:outline-none focus:bg-green-500 transition ease-in-out duration-150" do
+      link to: Breeze::Requests::Index, class: "#{sidebar_link_css} #{active_link_css_for_route(Breeze::Requests::Index)}" do
         tag "svg", class: "mr-3 h-6 w-6 text-green-200 group-hover:text-white group-focus:text-gray-600 transition ease-in-out duration-150", fill: "none", stroke: "currentColor", viewBox: "0 0 24 24" do
```

Comment:
@stephendolan you're pretty good with tailwind, so maybe you can help here. This SVG css currently assumes it's the active one. Here's what it looks like when it's not:

<img width="294" alt="Screen Shot 2021-02-21 at 10 05 07 AM" src="https://user-images.githubusercontent.com/2391/108634012-53e79100-742c-11eb-802d-1f332aaf9b9b.png">

This is on mouseover. Notice the icon disappears  
<img width="183" alt="Screen Shot 2021-02-21 at 10 05 13 AM" src="https://user-images.githubusercontent.com/2391/108634018-58ac4500-742c-11eb-8d88-cc122cbba4ae.png">

Reply:
Hey @jwoertink! I think it's just the `group-hover:text-white` that's making the icon look like it's disappearing. It's actually there, just really faint. I'd recommend probably just going with `group-hover:text-green-400/500` or something to just darken the green icon on hover.

I may be missing the issue, so just let me know! I don't have a dev env set up to pull this up yet, but happy to do so if it'd help.

Reply:
I am not 100% sure but I think I have a newer version of breeze in this branch with a much better design: https://github.com/paulcsmith/breeze-poc/tree/pcs/component-with-base-class. Or maybe a different branch 😂 It looks much nicer. Sorry I didn't bring this up sooner!